tests/test_dummy.py

```diff
@@ -117,9 +117,9 @@ def test_dummy_single_sweep_raw(name):
         averaging_mode=AveragingMode.CYCLIC,
         acquisition_type=AcquisitionType.RAW,
     )
-    results = platform.sweep(sequence, options, sweeper)
+    results = platform.execute([sequence], options, sweeper)
     assert pulse.id and pulse.qubit in results
-    shape = results[pulse.qubit].magnitude.shape
+    shape = results[pulse.qubit][0].magnitude.shape
```

Comment:
Following the above comment, why do we need to index using `[0]` here? This is probably a patch for tests to pass, however do we actually want the results to be a `dict` with `list` values now?

Reply:
This is just a patch, let's provide a better solution with #809 and #899